src/components/navigation.js

```diff
@@ -52,12 +53,19 @@ const Navigation = props => {
 	const { logoName, logoNavigateTo } = props
 	const { getTopLevelNavigation, getNonRulesNavigation } = data
 
+	const [isMenuShown, setIsMenuShown] = useState(true)
+	const toggleMenu = () => setIsMenuShown(!isMenuShown)
+
 	return (
-		<aside>
-			{/* className={this.state.showMenu ? 'show' : 'hide'} */}
-			{/* <button className="nav-hide-show-menu" onClick={e => this.handleHideShowMenu()}>
-        ☰
-    </button> */}
+		<aside className={isMenuShown ? 'show' : 'hide'}>
+			{/* hide menu when width <= 600px */}
+			<ReactMedia query="(max-width: 600px)" onChange={matches => matches && isMenuShown && toggleMenu()} />
```

Comment:
```suggestion
			<ReactMedia query="(max-width: 600px)" onChange={matches => matches && setIsMenuShown(false)} />
```
And being a type freak, I might even go as far as `matches => matches ? setIsMenuShown(false) : null` because `setIsMenuShown` (or `toggleMenu`) does not return a boolean…